client/components/seo/facebook-preview/index.jsx

```diff
@@ -10,9 +11,10 @@ import { compact } from 'lodash';
 import { firstValid, hardTruncation, shortEnough } from '../helpers';
 
 const TITLE_LENGTH = 80;
-const DESCRIPTION_LENGTH = 270;
+const DESCRIPTION_LENGTH = 200;
```

Comment:
Why does this need to change?..

Reply:
In my tests with longer post contents, I've noticed that FB cuts the text at _roughly_ 200 chars (actually something like 196, but for the sake of magic-number-clarity, I chose to be off of ~4 chars, which is still way better than being off of ~74 🙂).